### PR TITLE
Created DTO and Controller

### PR DESCRIPTION
**Pull request by RaoPrathyusha**

### CODE REVIEW

File: .idea/.gitignore
Comment (sid110055) [inline .idea/.gitignore:1]: remove .idea folder from here and add it in gitignore

File: CustomExceptionHandler.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestControllerAdvice
+public class CustomExceptionHandler {
```
Comment (sid110055) [inline CustomExceptionHandler.java:13]: change this CommonExceptionHandler

File: PersonController.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+
+@RestController
+@RequestMapping("/Person")
+@Validated
+
+public class PersonController {
+
+    @PostMapping
+    public ResponseEntity<Object> saveExample(@RequestBody @Valid Person person) {
```
Comment (sid110055) [inline PersonController.java:18]: do not give generic object as response, Use exact object what you're returning

File: src/main/java/com/expensecal/expensecal/controller/PersonController.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+import com.expensecal.expensecal.dto.Person;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/Person")
+
+public class PersonController {
```
Comment (sid110055) [inline src/main/java/com/expensecal/expensecal/controller/PersonController.java:12]: you can remove this class

File: src/main/java/com/expensecal/expensecal/dto/Person.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
+package com.expensecal.expensecal.dto;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+
+@Data
+    public class Person {
```
Comment (sid110055) [inline src/main/java/com/expensecal/expensecal/dto/Person.java:7]: format all the classes

**Comment by RaoPrathyusha** [comment]
Made all the changes as required.

**Review by sid110055 — APPROVED** [review]

**Pull request merged**